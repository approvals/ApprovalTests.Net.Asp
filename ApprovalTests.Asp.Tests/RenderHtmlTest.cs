# if DEBUG && !__MonoCS__
using ApprovalTests.Asp;
using ApprovalTests.Reporters;
using ApprovalTests.Scrubber;
using Asp.Net.Demo;
using Asp.Net.Demo.Orders;
using CassiniDev;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace ApprovalTests.Tests.Asp
{
	[TestClass]
	[UseReporter(typeof (DiffReporter), typeof (FileLauncherReporter))]
	public class RenderHtmlTest
	{
        private CassiniDevServer server = new CassiniDevServer();

        [TestInitialize]
        public void Setup()
        {
            PortFactory.AspPort = 1360;
            server.StartServer(Global.Path, PortFactory.AspPort, "/", "localhost");
        }

        [TestCleanup]
        public void TearDown()
        {
            server.StopServer();
        }

		[TestMethod]
		public void TestSimpleInvoice()
		{
			AspApprovals.VerifyAspPage(new InvoiceView().TestSimpleInvoice, HtmlScrubbers.ScrubAsp);

			//  -- These are the same thing
			//AspApprovals.VerifyUrl("http://localhost:1360/Orders/InvoiceView.aspx?TestSimpleInvoice");
		}

		[TestMethod]
		public void TestInternationalization()
		{
            AspApprovals.VerifyUrl("http://localhost:1360/Encoding.UTF8.html", HtmlScrubbers.ScrubBrowserLink);
		}
	}
}

#endif