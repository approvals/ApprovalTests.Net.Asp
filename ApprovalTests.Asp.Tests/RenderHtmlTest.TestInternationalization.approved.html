<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head><base href="http://localhost:1360">
   æ 
</head>
<body>
</body>
</html>
